moderator/settings.py

```diff
@@ -248,3 +248,24 @@ def _username_algo(email):
 
 # Enable dev login
 ENABLE_DEV_LOGIN = config("ENABLE_DEV_LOGIN", default=False, cast=bool)
+FROM_NOREPLY = config(
+    "FROM_NOREPLY",
+    default="No Reply - Mozilla Moderator <no-reply@moderator-mozilla.org",
```

Comment:
I think you're missing a `>` at the end here. And shouldn't it be `moderator.mozilla.org`?

I think standard practice is also to leave "no reply" out of the address name, i.e. `Mozilla Moderator <no-reply@moderator.mozilla.org>`